doc/user/content/integrations/terraform.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Terraform Materialize Provider"
+description: "Create and manage Materialize objects with the Terraform Materialize provider"
+menu:
+  main:
+    parent: "integrations"
+    weight: 11
+    name: "Terraform Materialize Provider"
+---
+
+Terraform is an infrastructure-as-code tool that allows you to manage your
+resources in a declarative configuration language. Materialize maintains a
+Terraform provider to help you configure Materialize resources like
+connections, sources, and database objects.
+
+## Use cases
+
+Materialize objects can be created through standard SQL queries, but managing
+more than a few components can be difficult to maintain. The Terraform
```

Comment:
Instead of the use case being for non-SQL users. Could it be something like "The Terraform Materialize provider allows users to manage the entire life cycle of their resources in a programmatic way. This can be helpful when managing your Materialize account alongside your other cloud resources."

Reply:
I like this framing and updated the text.

Do you think mentioning 'non-SQL' users comfortable with Terraform is a selling point for the provider? Curious if referencing HCL vs. SQL would be distracting.

Reply:
I don't think it is worth bringing up. I view HCL and SQL as just very different uses.